web/src/views/FindSpeaker.vue

```diff
@@ -96,19 +124,47 @@ export default {
     setError(err) {
       this.error = err;
     },
+    getPreviousPage() {
+      if (this.page > 1) {
```

Comment:
to avoid people messing with the pagination via the chrome inspector I add this additional safeguard here.

also here for the next page part:
https://github.com/WWCodeTokyo/speak-her-db/pull/166/files#diff-8d57168f5b367bbc58eeb3b2b3950debR160